.yeoman/templates/src/index.test.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import <%= componentName %> from './index';
+
+describe('<%= componentName %>', () => {
```

Comment:
Could we add an assertion on this instead of just having a snapshot test. If we have only a snapshot test, the developer creating this new package may decide to `updateSnapshots` and call it a day. Having assertions will encourage thoughtful testing.

Reply:
@sadick254 are you suggesting to write an assertion that will fail ?

like `expect(container.querySelector('h1').textContent).toEqual('Please write some valid tests');`

Reply:
@thekp Yes, thats much better